### PR TITLE
distance matrix departure time unit change from miliseconds to seconds

### PR DESCRIPTION
**Pull request by monajafi**
departure time changed from miliseconds to seconds

### CODE REVIEW

**Comment by monajafi** [comment]
I signed CLA

File: spec/e2e/distance-matrix-spec.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ var objectContaining = jasmine.objectContaining;
 describe('distance matrix client library', function() {
   var googleMaps = require('./service');
 
-  var inOneHour = new Date().getTime() + 60 * 60 * 1000;
+  var now = new Date().getTime();
+  var inOneHour = Math.round(now/1000);
```
Comment (stephenmcd) [inline spec/e2e/distance-matrix-spec.js]: is this still one hour ahead?

**Comment by monajafi** [comment]
Every time in this library is in milliseconds while google documentations says following about departure_time and arriavel_time:
departure_time — Specifies the desired time of departure. You can specify the time as an integer in seconds since midnight, January 1, 1970 UTC. Alternatively, you can specify a value of now, which sets the departure time to the current time (correct to the nearest second).
As you see it says time in seconds since january 1 ,1970 not milliseconds . you set milliseconds everywhere in tests not seconds. not just distanceMatrix I see in directions Api too

File: spec/e2e/distance-matrix-spec.js (hunk 1 above)
Comment (monajafi) [reply spec/e2e/distance-matrix-spec.js]: oh sorry you are right I've corrected it

**Pull request merged**